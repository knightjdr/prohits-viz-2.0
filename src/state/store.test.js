import React from 'react';
import * as moduleToMock from 'redux';
import { shallow } from 'enzyme';

import Store from './store';

// mock createStore
moduleToMock.createStore = () => ('');
jest.setMock('redux', moduleToMock);

describe('Store', () => {
  test('It renders', () => {
    const wrapper = shallow(
      <Store />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
