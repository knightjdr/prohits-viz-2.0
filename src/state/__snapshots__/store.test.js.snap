// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Store It renders 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <Routes />
</Provider>
`;

exports[`Store should render 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <Routes />
</Provider>
`;
